docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[network-beaconing-framework]]
+== Network Beaconing
+
+This feature provides an early warning system for command and control beaconing activity. It monitors network traffic for indicators of compromise and provides analytics to add context to alerts and aid your threat hunting.
+
+[discrete]
+=== Deploy the package
+
+To deploy the network beaconing framework in your environment using the https://github.com/elastic/detection-rules/releases/tag/ML-Beaconing-20211216-1[release package], follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+
+The release package includes dashboards for monitoring beaconing activity in your environment. Users can review signals via a Lens dashboard called `Network beaconing`.
+
+[role="screenshot"]
+image::images/beaconing-detection-1.png[]
+
+[discrete]
+=== Feature details
+
+The feature uses a {ref}/transforms.html[transform] to segment network data by host and process name, then runs scripted metric aggregations on the host-process name pairs. For a given time window, the scripted metric aggregation checks each partition for the following:
+
+* Signals repeating at regular intervals, accounting for minor variations in those intervals.
+* Low variation of bytes sent from source to destination.
+* Low variation of bytes sent from destination to source.
+
+The transform, which runs every hour, also filters out trusted applications and IPs in an attempt to reduce false positives. The transform outputs information about the detection and the process and host indicators, for example: 
+
+[role="screenshot"]
+image::images/beaconing-detection-2.png[]
+
+[discrete]
+=== Further customizations
+
+Advanced users can also tune the scripted metric aggregation's parameters, such as jitter percentage or time window. To overwrite the default parameters: delete the transform, change the parameters, and restart the transform. The configurable parameters are:
+
+* `number_buckets_in_range:` The number of time buckets into which the time window is split. Using more buckets will improve estimates for the various statistics, but also increase resource usage.
```

Comment:
Very small detail, but I'd put the colon outside the monospace backticks, so it doesn't look like part of the setting name. (I won't add the code suggestion to every line since that'd be noisy, just this first instance to illustrate how the others should also look.)
```suggestion
* `number_buckets_in_range`: The number of time buckets into which the time window is split. Using more buckets will improve estimates for the various statistics, but also increase resource usage.
```

Reply:
good call.